services/galley/src/Galley/Cassandra/SubConversation.hs

```diff
@@ -115,7 +122,13 @@ interpretSubConversationStoreToCassandra = interpret $ \case
   SetSubConversationEpoch cid sconv epoch -> embedClient $ setEpochForSubConversation cid sconv epoch
   DeleteGroupIdForSubConversation groupId -> embedClient $ deleteGroupId groupId
   ListSubConversations cid -> embedClient $ listSubConversations cid
-  DeleteSubConversation convId subConvId -> embedClient $ deleteSubConversation convId subConvId
+  DeleteSubConversation convId subConvId -> do
+    msub <- embedClient (selectSubConversation convId subConvId)
+    case msub of
+      Nothing -> embed @IO . UnliftIO.throwIO $ errorToWai @'ConvNotFound
```

Comment:
Why raising the exception and not making this call idempotent by just accepting that the intended state (delete conversation) has been reached?

Reply:
That's a fair point! I'll make this a no-op in case it is not found. I suppose this can happen when two concurrent requests to delete the same subconversation occur.

Reply:
@stefanwire , done in commit d904e392e.